lib/awful/wibar.lua

```diff
@@ -409,25 +592,37 @@ function awfulwibar.new(args)
 
     local w = wibox(args)
```

Comment:
@Elv13, what are the remaining missing pieces to implement before switching this to use popup?

Reply:
I am not sure. Definitively a lot of testing. The size changes due to widget size and the size changes due to screen geometry will most probably fight each other and need some mediation code?